quarkus-cli/src/test/resources/test.properties

```diff
@@ -1 +1,2 @@
 ts.global.generated-service.enabled=false
+ts.global.delete.folder.on.exit=false
```

Comment:
Why this line is need it? 

To us is important to remove the generated folders, in order to keep test suite memory "footprint" low.

Reply:
These folders are being cleaned during the next test. Do folders in target directory affect memory footprint?